webapp-django/crashstats/status/migrations/0002_auto_20180522_1545.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.11 on 2018-05-22 15:45
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('status', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='statusmessage',
+            name='severity',
+            field=models.CharField(choices=[(b'info', b'Info'), (b'warning', b'Warning'), (b'critical', b'Critical')], max_length=20),
+        ),
+    ]
```

Comment:
I moved the choices from the status message form to the model. That way they're available everywhere. That requires a migration.